next/components/styleguide/showcases/SelectFieldShowCase.tsx

```diff
@@ -1,59 +1,61 @@
 import React, { useState } from 'react'
 
 import SelectField from '../../forms/SelectField/SelectField'
-import SelectOption from '../../forms/SelectField/SelectOption'
+import SelectOption from '../../forms/SelectField/SelectRJSFOptions'
 import { Stack } from '../Stack'
 import { Wrapper } from '../Wrapper'
+import SelectRJSFOptions from '../../forms/SelectField/SelectRJSFOptions'
+import { EnumOptionsType } from '@rjsf/utils'
 
 interface SelectFieldShowCaseProps {
 
 }
 
 const SelectFieldShowCase = () => {
-  const selectOptions: SelectOption[] = [
-    { value: 'example', label: 'skola', description: 'this is testing'},
-    { value: 'stuFei', label: 'STU FEI', description: 'good choice'},
-    { value: 'stuFiit', label: 'STU FIIT'},
-    { value: 'ukFmfi', label: 'UK FMFI'},
-    { value: 'tukeFei', label: 'TUKE FEI' },
-    { value: 'unizaFeit', label: 'UNIZA FEIT' }
-  ]
+  const enumOptions: EnumOptionsType[] = [
+      { value: 'example', label: 'skola'},
+      { value: 'STU FEI', label: 'feika' },
+      { value: 'STU FIIT', label: 'fiitka' },
+      { value: 'UK FMFI', label: 'matfyz' },
+      { value: 'TUKE FEI', label: '' },
+      { value: 'UNIZA FEIT', label: 'UNIZA FEIT' }
+    ]
 
-  const [selectValueFirst, setSelectValueFirst] = useState<SelectOption[]>(selectOptions.slice(0, 2))
-  const [selectValueSecond, setSelectValueSecond] = useState<SelectOption[]>([])
-  const [selectValueThird, setSelectValueThird] = useState<SelectOption[]>(selectOptions.slice(0, 3))
-  const [selectValue4, setSelectValue4] = useState<SelectOption[]>([])
-  const [selectValue5, setSelectValue5] = useState<SelectOption[]>(selectOptions.slice(2, 3))
-  const [selectValue6, setSelectValue6] = useState<SelectOption[]>([])
+  const [selectValueFirst, setSelectValueFirst] = useState<EnumOptionsType[]>(enumOptions.slice(0, 2))
```

Comment:
I wanted to show that I handled also case like this, where programmer would put this to type="one". Its not possible in rjsf but if somebody would use it, its handled and it will behave like its chosen only first. but i can chage it like to want in styleguide.